PATCH-P1-FIX-UPSTREAM: Fix mysql_install_db to find data files in mariadb directories

Fix mysql_install_db.sh script to find data files in mariadb directories
when a user uses "--basedir" option [bsc#1006539]

Maintainer: Kristyna Streitova <kstreitova@suse.com>

Index: mariadb-10.2.3/scripts/mysql_install_db.sh
===================================================================
--- mariadb-10.2.3.orig/scripts/mysql_install_db.sh
+++ mariadb-10.2.3/scripts/mysql_install_db.sh
@@ -289,16 +289,16 @@ then
     cannot_find_file mysqld $basedir/libexec $basedir/sbin $basedir/bin
     exit 1
   fi
-  langdir=`find_in_basedir --dir errmsg.sys share/english share/mysql/english`
+  langdir=`find_in_basedir --dir errmsg.sys share/english share/mysql/english share/mariadb/english`
   if test -z "$langdir"
   then
-    cannot_find_file errmsg.sys $basedir/share/english $basedir/share/mysql/english
+    cannot_find_file errmsg.sys $basedir/share/english $basedir/share/mysql/english $basedir/share/mariadb/english
     exit 1
   fi
-  pkgdatadir=`find_in_basedir --dir fill_help_tables.sql share share/mysql`
+  pkgdatadir=`find_in_basedir --dir fill_help_tables.sql share share/mysql share/mariadb`
   if test -z "$pkgdatadir"
   then
-    cannot_find_file fill_help_tables.sql $basedir/share $basedir/share/mysql
+    cannot_find_file fill_help_tables.sql $basedir/share $basedir/share/mysql $basedir/share/mariadb
     exit 1
   fi
   scriptdir="$basedir/scripts"
